bus-mapping/src/circuit_input_builder.rs

```diff
@@ -352,12 +353,21 @@ impl<P: JsonRpcClient> BuilderClient<P> {
     }
 
     /// Perform all the steps to generate the circuit inputs
-    pub async fn gen_inputs(&self, block_num: u64) -> Result<CircuitInputBuilder, Error> {
```

Comment:
See other comment regarding `get_block`. If we don't consume both `builder` and `eth_block` then we can also just leave this unchanged und use `get_block`

Reply:
I agree with the idea. I just noticed that (at the moment) only one item of the result is used in the test so I was wondering about that.